fw/http_parser.c

```diff
@@ -12311,6 +12331,11 @@ tfw_http_parse_resp(void *resp_data, unsigned char *data, unsigned int len,
 				   __parse_content_length,
 				   TFW_HTTP_HDR_CONTENT_LENGTH);
 
+	/* 'Content-Location:*OWS' is read, process field-value. */
+	TFW_HTTP_PARSE_SPECHDR_VAL(Resp_HdrContent_LocationV, msg,
+				   __parse_content_location,
+				   TFW_HTTP_HDR_CONTENT_LOCATION);
```

Comment:
Please update `HP_ENTRY_NAME` for this header in hpack.c.

Reply:
According to our private discussion we don't need this, because we don't parse this header for requests